_install-and-configure/upgrade-opensearch/appendix/rolling-upgrade-lab.md

```diff
@@ -0,0 +1,887 @@
+   cb5188308b21   opensearchproject/opensearch:1.3.7              "./opensearch-docker…"   25 seconds ago   Up 24 seconds   9300/tcp, 9650/tcp, 0.0.0.0:9204->9200/tcp, :::9204->9200/tcp, 0.0.0.0:9604->9600/tcp, :::9604->9600/tcp   os-node-04
+   71b682aa6671   opensearchproject/opensearch:1.3.7              "./opensearch-docker…"   26 seconds ago   Up 25 seconds   9300/tcp, 9650/tcp, 0.0.0.0:9203->9200/tcp, :::9203->9200/tcp, 0.0.0.0:9603->9600/tcp, :::9603->9600/tcp   os-node-03
+   f894054a9378   opensearchproject/opensearch:1.3.7              "./opensearch-docker…"   27 seconds ago   Up 26 seconds   9300/tcp, 9650/tcp, 0.0.0.0:9202->9200/tcp, :::9202->9200/tcp, 0.0.0.0:9602->9600/tcp, :::9602->9600/tcp   os-node-02
+   2e9c91c959cd   opensearchproject/opensearch:1.3.7              "./opensearch-docker…"   28 seconds ago   Up 27 seconds   9300/tcp, 9650/tcp, 0.0.0.0:9201->9200/tcp, :::9201->9200/tcp, 0.0.0.0:9601->9600/tcp, :::9601->9600/tcp   os-node-01
+   ```
+1. The amount of time OpenSearch needs to initialize the cluster varies depending on the performance capabilities of the underlying host. You can watch container logs to see what OpenSearch is doing during the bootstrap process:
+   1. Enter the following command to display logs for container `os-node-01` in the terminal window:
+      ```bash
+      docker logs -f os-node-01
+      ```
+      {% include copy.html %}
+   1. You will see a log entry like the following example when the node is ready:
+      <p class="codeblock-label">Example</p>
+      ```
+      [INFO ][o.o.s.c.ConfigurationRepository] [os-node-01] Node 'os-node-01' initialized
+      ```
+   1. Press `Ctrl+C` to stop following container logs and return to the command prompt.
+1. Use cURL to query the OpenSearch REST API. In the following command, `os-node-01` is queried by sending the request to host port `9201`, which is mapped to port `9200` on the container:
+   ```bash
+   curl -s "https://localhost:9201" -ku admin:admin
+   ```
+   {% include copy.html %}
+   <p class="codeblock-label">Example response</p>
+   ```json
+   {
+       "name" : "os-node-01",
+       "cluster_name" : "opensearch-dev-cluster",
+       "cluster_uuid" : "g1MMknuDRuuD9IaaNt56KA",
+       "version" : {
+           "distribution" : "opensearch",
+           "number" : "1.3.7",
+           "build_type" : "tar",
+           "build_hash" : "db18a0d5a08b669fb900c00d81462e221f4438ee",
+           "build_date" : "2022-12-07T22:59:20.186520Z",
+           "build_snapshot" : false,
+           "lucene_version" : "8.10.1",
+           "minimum_wire_compatibility_version" : "6.8.0",
+           "minimum_index_compatibility_version" : "6.0.0-beta1"
+       },
+       "tagline" : "The OpenSearch Project: https://opensearch.org/"
+   }
+   ```
+
+**TIP**: Use the `-s` option with `curl` to hide the progress meter and error messages.
```

Comment:
Do we need the bold **TIP** here? It does make it easier to see in the local build, so I'm okay keeping it, but it might be overkill. 

```suggestion
Use the `-s` option with `curl` to hide the progress meter and error messages.
```

Reply:
Yes - it looks bland without it, and a reader has no awareness of what a color border means (i.e. what gets added by `{: .tip}`).